l10n_de_country_states/README.rst

```diff
@@ -1,22 +1,46 @@
+=====================
```

Comment:
Include license flag as in the README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst